devices/Ft6xx6x/samples/Program.cs

```diff
@@ -1,55 +1,226 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-
-using System.Device.Gpio;
-using System.Device.I2c;
+using System;
 using System.Diagnostics;
 using System.Threading;
-using Iot.Device.Ft6xx6x;
 using nanoFramework.M5Stack;
+using nanoFramework.M5Core2;
+using Iot.Device.Button;
+using Iot.Device.Ft6xx6x;
+using System.Device.Gpio;
+using System.Device.I2c;
```

Comment:
Please remove the using not needed. So it makes things easier to read.

Reply:
This is your original code https://github.com/nanoframework/nanoFramework.IoT.Device/blob/develop/devices/Ft6xx6x/samples/Program.cs  ...i dont understand it.....and please allow me to post the whole content of Program.cs 1:1 as it compiles , build and deploy  on my machine...if we just post incomplete code it makes extremely difficult to impossible for a newbie and even for an experience developer to jump start in the NF .....if i delete some of the using the code will not work at once...there is already enough hurdles to start working with the NF ie..finding the right NuGets/Version/Dependencies....please!

Reply:
Removing an unused using line doesn't harm anything... not getting your point here... 🤔 
You can easily pick those in VS as they'll show in a dim color to highlight that. Plus you can use the VS command "remove and reorder usings".